src/components/Menus/Sidebar/InstructionsPanel/InstructionsPanel.jsx

```diff
@@ -128,13 +132,33 @@ const InstructionsPanel = () => {
       <div className="project-instructions">
         {instructionsEditable ? (
           hasInstructions ? (
-            <div>
+            <div className="c-instruction-tabs">
               {instructionsEditable && (
-                <textarea
-                  data-testid="instructionTextarea"
-                  value={project.instructions}
-                  onChange={onChange}
-                ></textarea>
+                <Tabs
+                  onSelect={(index) => {
+                    setInstructionsTab(index);
+                  }}
+                >
+                  <TabList>
+                    <Tab>{t("instructionsPanel.edit")}</Tab>
+                    <Tab>{t("instructionsPanel.view")}</Tab>
+                  </TabList>
+                  <TabPanel>
+                    <textarea
+                      data-testid="instructionTextarea"
+                      value={project.instructions}
+                      onChange={onChange}
+                    ></textarea>
+                  </TabPanel>
+                  <TabPanel>
+                    <>
+                      <div
+                        className="project-instructions"
+                        ref={stepContent}
+                      ></div>
+                    </>
+                  </TabPanel>
+                </Tabs>
```

Comment:
Do you think it's worth having a couple of tests for these changes?

Reply:
Fixed in [1a39c0b](https://github.com/RaspberryPiFoundation/editor-ui/pull/1167/commits/1a39c0b799ec46e0b579f706d194e73a86554565)